robottelo/common/helpers.py

```diff
@@ -151,6 +151,32 @@ def generate_strings_list(len1=None):
     ]
 
 
+def generate_strings_dict(len1=None):
```

Comment:
Now, all the above methods have been merged and we now only have `generate_strings_list()` method.